doc/ref_cert/RC1/chapters/chapter04.md

```diff
@@ -91,7 +91,7 @@ different pools of virtual machines.
 
 Here are the functional tests (>2000) running in OpenStack gates integrated in
 Functest Smoke (see
-[Functest daily jobs](https://build.opnfv.org/ci/job/functest-hunter-daily/392/)
+[Functest daily jobs](https://build.opnfv.org/ci/view/functest/job/functest-jerma-daily/371/)
```

Comment:
Why are we referencing a specific functest run instead of, say, https://build.opnfv.org/ci/job/functest-jerma-daily/; at the bottom, one can see the latest successful and unsuccessful runs.

Reply:
@pgoyal01 I woud consider fixed links as better here.  Why not, but I would do it in another PR modifying all links in RA and RC.